### ISSUE TITLE
FEATURE REQUEST: Make is super easy to send ZANO between local wallets via a drop-down wallet selector in the UI

### ISSUE THREAD

**Issue by mvozzo** [hyle-team/zano_ui#122]
If a user has multiple wallets in the Zano GUI Desktop wallet, instead of making the users to copy-n-paste addresses to send between these local wallets, please allow for a dropdown allowing the user to simply select a local wallet to send to.  

The dropdown will show the Wallet's "friendly" name (as displayed in the left hand wallet selector column). 

**Why is this important?**
1) It removes any possible human error with copy-n-paste of wallet addresses
2) It removes any potential spyware that may monitoring clipboard activity, thus compromising the user's wallet address and potentially doxing them by linking to their identity, and most importantly, 
3) It will be a much easier and simpler user experience to transfer funds between local wallets.

**Comment by cryptozoidberg** [comment]
@PRavaga can you look into this?

**Comment by PRavaga** [comment]
Yeah that totally makes sense and I agree. There were few more requests to add this feature as well. So it's on the list and will be implemented in one of the following releases.
